README.md

```diff
@@ -1,28 +1,38 @@
 <!--- Top of README Badges (automated) --->
 [![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/icecube/skymap_scanner?include_prereleases)](https://github.com/icecube/skymap_scanner/) [![Lines of code](https://img.shields.io/tokei/lines/github/icecube/skymap_scanner)](https://github.com/icecube/skymap_scanner/) [![GitHub issues](https://img.shields.io/github/issues/icecube/skymap_scanner)](https://github.com/icecube/skymap_scanner/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen) [![GitHub pull requests](https://img.shields.io/github/issues-pr/icecube/skymap_scanner)](https://github.com/icecube/skymap_scanner/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aopen) 
 <!--- End of README Badges (automated) --->
-# skymap_scanner
+# Skymap Scanner v3
 
 A distributed system that performs a likelihood scan of event directions for real-time alerts using inter-CPU queue-based message passing.
 
+Skymap Scanner is the computational core of the [SkyDriver orchestration service](https://github.com/WIPACrepo/SkyDriver).
+
 `skymap_scanner` is a python package containing two distinct applications meant to be deployed within containers (1 `skymap_scanner.server`, n `skymap_scanner.client`s), along with `skymap_scanner.utils` (utility functions) and `skymap_scanner.recos` (`icetray` reco-specific logic). Additional, package-independent, utility scripts are in `scripts/utils/`.
 
 ### Example Startup
 You will need to get a pulsar broker address and authentication token to pass to both the server and client. Send a poke on slack #skymap-scanner to get those!
 
 #### 1. Launch the Server
-The server can be launched from anywhere with a stable network connection. You can run it from the cobalts for example. For now, set `--timeout-to-clients` and `--timeout-from-clients` to a large value like 300000. This will persist the server in case the clients don't start up right away.
+The server can be launched from anywhere with a stable network connection. You can run it from the cobalts for example.
+
+###### In-Production Usage Note: timeout values
+For now, set `SKYSCAN_MQ_TIMEOUT_TO_CLIENTS` and `SKYSCAN_MQ_TIMEOUT_FROM_CLIENTS` to a large value like 300000. This will persist the server in case the clients don't start up right away.
+
 ##### Figure Your Args
+###### Environment Variables
+```
+export SKYSCAN_BROKER_AUTH=$(cat ~/skyscan-broker.token)  # obfuscated for security
```

Comment:
Is the broker auth tied to a particular broker address? If so it might not make sense to separate the two, otherwise it's fine.

Reply:
> Is the broker auth tied to a particular broker address

Reply:
> Is the broker auth tied to a particular broker address
Yes, as in the broker auth token is specific to a broker instance.

I could make the broker address an env var as well. My main objective is to put sensitive strings in env vars. The address is not necessarily sensitive info. But I see your point.

Reply:
Seeing that you're our only manual user :smile: I've made them both env vars